src/media/css/feed/feed.styl

```diff
@@ -71,11 +71,15 @@
     }
     .quote {
         color: $white;
-        left: -10px;
+        left: -5px;
         padding: 5px 0 15px;
         position: relative;
         top: 15px;
         width: $mobile-tile-width;
+
+        +rtl() {
+            left: 10px;
```

Comment:
Seeing this 5px off also, 15px works for me.

![screenshot 2015-04-23 09 31 04](https://cloud.githubusercontent.com/assets/211578/7299444/7d900450-e99b-11e4-89fc-1bf312898a3e.png)